### ISSUE TITLE
Deno install - PATH isn't added

### ISSUE THREAD

**Issue by MatteoGauthier** [denoland/deno#5328]
After executing deno install v1.0.0 on windows. The command is unavailable. Then when i add `C:\Users\Mattaio\.deno\bin` to the path the command remain unavailable because of the extension .cmd

**Comment by Julienboyreau** [comment]
Maybe same problem here
When typing "deno --version" in the executing downloaded deno.exe, I have the following error message.

`> deno --version
Uncaught SyntaxError: Unexpected identifier
    at ←[1m←[3m←[90mevaluate←[39m←[23m←[22m ←[90m(←[39m←[90m$deno$/repl.ts←[39m←
[90m:←[39m←[90m45←[39m←[90m:←[39m←[90m34←[39m←[90m)←[39m
    at ←[1m←[3m←[90mObject.replLoop←[39m←[23m←[22m ←[90m(←[39m←[90m$deno$/repl.t
s←[39m←[90m:←[39m←[90m136←[39m←[90m:←[39m←[90m13←[39m←[90m)←[39m`

**Comment by DaemonicNaymon** [comment]
I got it working by doing the following:

1. go to deno.land
I went to the section installation and picked powershell (windows)

2. open up powershell (just type powershell in windows searchbar)

3. copy iwr https://deno.land/x/install/install.ps1 -useb | iex (which i got by doing the above) into the powershell

after letting powershell doing its work you will get the following.

![image](https://user-images.githubusercontent.com/24189525/82088710-853f6d80-96f2-11ea-96d5-217beef2fd62.png)

4. startup cmd or powershell again and type in deno --version and it should give you the correct output.
